modules/Bio/EnsEMBL/Slice.pm

```diff
@@ -2461,33 +2460,6 @@ sub get_by_Individual{
 
 }
 
-=head2 get_by_strain
```

Comment:
So this should have been removed quite well before.

Reply:
Yes. Everything Individual related can go. I haven't found any dependencies in the web code. I will deprecate the Individual code in a separate PR.

Reply:
That makes sense, thanks.